content/source/docs/enterprise/saml/identity-provider-configuration-adfs.html.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: enterprise2
+page_title: "ADFS Configuration - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-private-saml-identity-provider-configuration-adfs"
+---
+
+# ADFS Configuration
+
+This guide explains how to configure Active Directory Federated Services (ADFS) in order to use it as an Identity Provider (IdP) for PTFE's SAML authentication feature. The screenshots below were taken on Windows Server 2016, and the UI may not look the same on previous Windows versions.
+
+This document assumes that you have already installed and configured ADFS and that you are using PTFE version v201807-1 or later.
```

Comment:
Is this mostly about using the new SAML settings endpoint? I wonder how we should handle this...the OneLogin one is older; should we update that, or move this one back to v1, or...?

Reply:
This is a guide on how to configure SAML when your IdP is MS ADFS end to end. The fact that the metadata URL is there was just an added bonus.

I think it makes more sense to update the OneLogin one too.

Reply:
Yeah, understanding that you're using the metadata URL, I agree. I'll write a quick task to remind us to do that at some point.